### DISCUSSION TITLE
HWUI for Avalonia?

### DISCUSSION THREAD

**Discussion by byme8** [AvaloniaUI/Avalonia#8693] [Q&A] [answered]
Does Avalonia have something similar to Android HWUI? https://developer.android.com/topic/performance/rendering/inspect-gpu-rendering 
It can be really handy during performance issues investigation.

**Comment by maxkatz6** [comment, marked as answer]
We don't have anything powerful like HWUI in android. Instead, you have the usual .NET profiling tools (dotTrace, VisualStudio...). Although I can agree it's not as useful on mobile platforms.
